go.mod

```diff
@@ -2,28 +2,29 @@ module github.com/jjliggett/jjversion-gha-output
 
 go 1.19
 
-require github.com/jjliggett/jjversion v0.5.18
+require github.com/jjliggett/jjversion v0.5.20
```

Comment:
This relates to the following changes to `jjversion`: https://github.com/jjliggett/jjversion/pull/174 and https://github.com/jjliggett/jjversion/pull/180